Jarvis-Platform/jarvis-studio
# Feature/generic vm launcher

[comment]: [![Status](https://img.shields.io/badge/Status-DEVELOPMENT-orange.svg)]()
[comment]: [![Status](https://img.shields.io/badge/Status-HOLD-inactive.svg)]()

[![Status](https://img.shields.io/badge/Status-READY-brightgreen.svg)]()


## Description
Use generic components for `vm-launcher` views


### Runs
- [x] Use `listing-component` for `ListingView`
- [x] Create `ItemView` + route & use `item-component`

### Configurations
- [x] Use `listing-component` for `ListingView`
- [x] Create `ItemView` + route & use `item-component`


## Deploy Notes
Review + Merge on master


## Steps to Test or Reproduce
```sh
git checkout master
git pull
git checkout feature/generic-vm-launcher
```

1. Go to `/data-workflows/vm-launcher/runs` => Listing should be ok
2. Go to `/data-workflows/vm-launcher/runs/:id` => Item informations should be ok
3. Go to `/data-workflows/vm-launcher/configurations` => Listing should be ok
4. Go to `/data-workflows/vm-launcher/configurations/:id` => Item informations should be ok


## Impacted Areas in Application
* `/data-workflows/vm-launcher/*` pages
